test/devapps/authorization_code_sample.go

```diff
@@ -41,10 +42,10 @@ func getToken(w http.ResponseWriter, r *http.Request) {
 	}
 	code := codes[0]
 	// Getting the access token using the authorization code
-	authCodeParams := msal.CreateAcquireTokenAuthCodeParameters(config.Scopes, config.RedirectURI)
-	authCodeParams.Code = code
-	authCodeParams.CodeChallenge = config.CodeChallenge
-	result, err := publicClientApp.AcquireTokenByAuthCode(authCodeParams)
+	result, err := publicClientApp.AcquireTokenByAuthCode(context.Background(), config.Scopes, config.RedirectURI, &msal.AcquireTokenByAuthCodeOptions{
```

Comment:
Is it always the same URL?  What about for Azure Stack and/or sovereign clouds?

Reply:
No it's not always the same URL The URL depends on the cloud, but this could be part of the options? (most customers would use the public Azure cloud?)

Reply:
I see.  So just to confirm, we always specify this same redirect URL but since we never follow it there's no need to make it configurable, is that correct?

Reply:
The `redirectURI` param has been removed.

Reply:
@jmprieur I've removed the param, anything else or is this good to merge?